src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -0,0 +1,113 @@
+package com.google.sps.servlets;
+
+import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.io.*;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Collections;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles forum data (questions, comments, and replies) */
+@WebServlet("/account")
```

Comment:
I feel like we should try and have the servlet URL and the name of the class be consistent, so I think we should either rename this as `@WebServlet("/user_data")` or we should rename the class to `AccountServlet`.

The benefit is that it makes it easier for someone in the future (often not you) to read/debug the code. If they see a request to `/user_data` and code for a servlet called `UserDataServlet` it's much easier to make the connection without digging into the code itself.

Reply:
hmmmm can we change this once everything is pushed up to master? We use /account in multiple places, so I think we should get all the working code on master before we make this fix